src/radical/pilot/pilot_manager.py

```diff
@@ -198,7 +193,7 @@ def close(self, terminate=True):
         self._terminate.set()
         self._rep.info('<<close pilot manager')
 
-        # we don't want any callback invokations during shutdown
+        # disable callbacks during shutdown
         # FIXME: really?
```

Comment:
Found this funny. Why really?

Reply:
In one way, I don't want to get notified ' you are terminating' when I initiate a termination - that is redundant.  And we need to somewhat carefully check that we don't run circles during termination (callback encountering termination and thus trying to terminate...).  Other than that I am not sure - we should eventually revisit that.

Thanks for the typos, will fix.